_data/projects/mellium.yml

```
@@ -0,0 +1,12 @@
+---
+name: Mellium
+desc: An implementation of XMPP (Jabber) in Go.
+site: https://mellium.im
+tags:
+- go
+- jabber
+- xmpp
+- im
+upforgrabs:
+  name: good first issue
+  link: https://github.com/mellium/xmpp/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22
```

The label itself is a bit more succinct:

```suggestion
  link: https://github.com/mellium/xmpp/labels/good%20first%20issue
```

---

Done; thanks.